src/widgets/histogram/content-view.js

```diff
@@ -255,8 +254,7 @@ module.exports = cdb.core.View.extend({
 
   _onBrushEnd: function (loBarIndex, hiBarIndex) {
     var data = this._dataviewModel.getData();
-
-    if (!data || !data.length) {
+    if ((!data || !data.length) || (this.model.get('lo_index') == loBarIndex && this.model.get('hi_index') == hiBarIndex)) {
```

Comment:
Avoid executing the function if the new range is the same as the one set previously